descqa/StellarMassFunction.py

```diff
@@ -0,0 +1,353 @@
+from __future__ import print_function, division, unicode_literals, absolute_import
+import os
+import math
+try:
+    from itertools import zip_longest
+except ImportError:
+    from itertools import izip_longest as zip_longest
+
+import numpy as np
+from GCR import GCRQuery
+
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+from itertools import cycle
+from itertools import count
+
+__all__ = ['StellarMassFunction']
+
+
+class StellarMassFunction(BaseValidationTest):
+    """
+    validation test to show N(z) distributions
+    """
+    #setup dict with parameters needed to read in validation data
+    possible_observations = {
+        'PRIMUS_2013': {
+            'filename_template': 'SMF/moustakas_et_al_2013/Table{}.txt',
+            'file-info': {'0. < z < .1':{'zlo':0., 'zhi':.1, 'table#':3, 'usecols':[0,1,2,3]},
+                        '.2 < z < .3':{'zlo':.2, 'zhi':.3, 'table#':4, 'usecols':[0,1,2,3]},
```

Comment:
minor but aligning the first quote symbol in this and following lines would make this look nicer

Reply:
Done.